computeengine/mailjet.js

```diff
@@ -18,7 +18,8 @@ var mailer = require('nodemailer');
 var smtp = require('nodemailer-smtp-transport');
 
 var transport = mailer.createTransport(smtp({
-  service: 'Mailjet',
+  host: "in.mailjet.com",
```

Comment:
done!

On Tue, Jun 14, 2016 at 11:39 AM, Jason Dobry <notifications@github.com>
wrote:

> In computeengine/mailjet.js
> <https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/125#discussion_r67007784>
> :
>
> > @@ -18,7 +18,8 @@ var mailer = require('nodemailer');
> >  var smtp = require('nodemailer-smtp-transport');
> >
> >  var transport = mailer.createTransport(smtp({
> > -  service: 'Mailjet',
> > +  host: "in.mailjet.com",
>
> Can you please change this from double quotes to single quotes? The
> linting will not pass unless this uses single quotes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/125/files/3227606f313fda9a993eb8fb956a3299bc880cda#r67007784>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AOZHMMyO33ZemR3xQIvhEqmWiaf7-APMks5qLtk9gaJpZM4I0klC>
> .
>



-- 
Chris Roed - Sr Systems Engineer

Anser Innovation LLC

13963 W. Preserve Blvd  #200

Burnsville, MN 55337



chris.roed@anserinnovation.com <lmlavin@anserinnovation.com>

www.anserinnovation.com

www.petchatz.com


<http://www.anserinnovation.com/> <http://www.anserinnovation.com/>Pioneering
technologies that enable '*being there from anywhere'*


Confidentiality Notice: The documents accompanying this email and the
content of this email contain confidential information belonging to the
sender, which is legally privileged. This information is intended for the
use of the individual or entity named above. If you are not the intended
recipient, you are hereby notified that any disclosure, copying,
distribution or the taking of any action in reliance on or regarding the
content of this email is strictly prohibited.